### DISCUSSION TITLE
More detailed Analytics

### DISCUSSION THREAD

**Discussion by s2erge** [modrinth/.github#180] [Feature requests]
As of right now it's confusing and quite forgetful regarding the revenue analytics!

I would propose a record on how much did we earn by each day, so we don't need to use external tools to calculate that like an excel sheet!

I'm not entirely sure why there is a total revenue that counts even the gains that were withdrawn, but it would be highly helpful having at least one that tells how much you had in the previous months, like records!

**Comment by triphora** [comment]
This is already on [the roadmap](https://roadmap.modrinth.com), suggestions need not be created for anything on the roadmap.
